src/map/clif.cpp

```diff
@@ -1007,6 +1007,7 @@ void clif_clearunit_delayed(struct block_list* bl, clr_type type, t_tick tick)
 {
 	struct block_list *tbl = ers_alloc(delay_clearunit_ers, struct block_list);
 	memcpy (tbl, bl, sizeof (struct block_list));
```

Comment:
```suggestion
	tbl->next = nullptr;
	tbl->prev = nullptr;
	tbl->id = bl->id;
	tbl->m = bl->m;
	tbl->x = bl->x;
	tbl->y = bl->y;
```
To get rid of another memcpy and to also make sure next/prev are not being used.